### ISSUE TITLE
Should emit the HTTP status line in response output

### ISSUE THREAD

**Issue by MikeTaylor** [thefrontside/okapi.rb#14]
If I do this:
```
$ cat ExternalDeploymentDescriptor.json | okapi create --no-tenant /_/discovery/modules
{
  "instId": "127.0.0.1-3000",
  "srvcId": "folio_graphql-0.1.0",
  "url": "http://127.0.0.1:3000"
}
$
```
The response to the (presumably) successful HTTP request is shown, but no explicit indication of the status. I would find it helpful to see "200 OK" before the response body -- or, at least, to have a command-line option that allows this.

**Comment by cowboyd** [comment]
Right now we kind of cheat by just saying if it's a `String`, pass it along, otherwise treat it as JSON and pretty print it.

We'd need some mechanism to allow the subcommand so provide "sections" of output. Almost like layout, but not quite maybe? It's actually a bit of a head-scratcher.

**Comment by MikeTaylor** [comment]
I'm not seeing the difficulty. What I suggested here was not a change in how the response body is handled -- what you're doing now seems like a great example of getting 99% of "perfect" functionality for 1% of the effort. All I'm saying is that before that's emitted, it would be good to also get to "200 OK" or "404 Not found" or what have you from the HTTP response status.

**Comment by cowboyd** [comment]
Yeah, maybe my wishy-washiness is me overthinking it. The thing I'm fiddling with in my mind is that the code responsible for making the request is not the code responsible for printing to the console, so by the time it gets to the print-y portion, there's no concept that this was a request (as opposed to a configuration command).

We could just represent it in the model, but that would give you something like:

```json
{
 "status": "201 Created",
 "body": {
    "instId": "127.0.0.1-3000",
    "srvcId": "folio_graphql-0.1.0",
    "url": "http://127.0.0.1:3000"
  }
}
```

But I think that it would maybe look better as:

```
201 Created
{
  "instId": "127.0.0.1-3000",
  "srvcId": "folio_graphql-0.1.0",
  "url": "http://127.0.0.1:3000"
}
```

Again, maybe I'm overthinking it?

**Comment by MikeTaylor** [comment]
Got to admit, I've not looked at the code _at all_ -- happy to trust you guys with it! But as a user, the latter output is the one I would expect, as a more honest statement of what just happened.
